Docker/start-latest.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
This script is run with `bash` by the docker file, maybe use `/bin/bash` here for consistency or run with `sh`?

Reply:
Should be fixed. (957dc6e8f882517049be136094b12fd557143da5)